runtime/basilisk/src/xcm.rs

```diff
@@ -57,15 +57,17 @@ pub type LocalOriginToLocation = SignedToAccountId32<RuntimeOrigin, AccountId, R
 
 pub type Barrier = (
 	TakeWeightCredit,
-	AllowTopLevelPaidExecutionFrom<Everything>,
 	// Expected responses are OK.
 	AllowKnownQueryResponses<PolkadotXcm>,
-	// Subscriptions for version tracking are OK.
-	AllowSubscriptionsFrom<Everything>,
-	// Tinkernet Multisigs require DescendOrigin, this barrier makes sure it's followed by payment.
-	// The barrier will only pass when the origin is a Tinkernet multisig.
 	WithComputedOrigin<
-		AllowTopLevelPaidExecutionFrom<invarch_xcm_builder::TinkernetMultisigMultiLocation>,
+		(
+			AllowTopLevelPaidExecutionFrom<Everything>,
+			// Subscriptions for version tracking are OK.
+			AllowSubscriptionsFrom<Everything>,
+			// Tinkernet Multisigs require DescendOrigin, this barrier makes sure it's followed by payment.
+			// The barrier will only pass when the origin is a Tinkernet multisig.
+			AllowTopLevelPaidExecutionFrom<invarch_xcm_builder::TinkernetMultisigMultiLocation>,
+		),
 		UniversalLocation,
 		ConstU32<1>,
```

Comment:
Same comment as for Hydra, ecosystem seems to go for 8, no reason to deviate

Reply:
@Roznovjak ?

Reply:
done